test/Atmos/EDMF/stable_bl_single_stack_implicit.jl

```diff
@@ -212,8 +215,8 @@ function main(::Type{FT}) where {FT}
         δρe = (weightedsum(Q, ρe_idx) .- Σρe₀) ./ Σρe₀
         @show (abs(δρ))
         @show (abs(δρe))
-        @test (abs(δρ) <= 1.0e-10)
-        @test (abs(δρe) <= 0.025)
+        @test (abs(δρ) <= 0.001)
+        @test (abs(δρe) <= 0.1)
```

Comment:
not sure you need this change before you are running implicit - and as long as it is not implicit in master maybe we don't need to change it ?

Reply:
this is needed for the explicit case as well.